python/gps/algorithm/cost/cost_fk.py

```diff
@@ -51,10 +51,16 @@ def eval(self, sample):
         tgt = self._hyperparams['target_end_effector']
         pt = sample.get(END_EFFECTOR_POINTS)
         dist = pt - tgt
-        # TODO - These should be partially zeros so we're not double
-        #        counting.
-        #        (see pts_jacobian_only in matlab costinfos code)
-        jx = sample.get(END_EFFECTOR_POINT_JACOBIANS)
+        if self._hyperparams.get('use_jacobian', True):
```

Comment:
I put this change back since if we extract samples from saved pickle files, the jacobians are missing.